src/openms/include/OpenMS/FORMAT/FASTAFile.h

```diff
@@ -38,187 +38,175 @@
 #include <OpenMS/DATASTRUCTURES/String.h>
 #include <OpenMS/CONCEPT/ProgressLogger.h>
 
-#include <functional> // for std::function
+#include <functional>
 #include <fstream>
-#include <memory>  // for unique_ptr
+#include <memory>
+#include <utility>
```

Comment:
you can probably remove `functional` and `memory` includes since they were only useful for the unique_ptr (which is gone now)